### PR TITLE
ranges: fix referencing stale player objects

### PR DESCRIPTION
**Pull request by SJShaw**

### CODE REVIEW

File: ranges.lua [hunk 1]
```diff
@@ -178,36 +178,41 @@ function disableRangeSmokeRefresh(rangeGroup)
...
   if rangeGroupName ~= nil then
     local rangeGroup = Group.getByName(rangeGroupName)
     if rangeGroup then
       rangeGroup:destroy()
-      log("Destroying range group")
+      TNN.log("Destroying range group")
     end
     if range["jtacGroup"] then
-      log("Range had jtac. Destroying.")
+    TNN.log("Range had jtac. Destroying.")
```
Comment (Lugghawk) [inline ranges.lua]: Indentation here doesn't look right on the review. Are you using the right whitespace chars?

File: ranges.lua [hunk 2]
```diff
@@ -374,9 +389,11 @@ function rangeDeathCheck()
       if not HOGGIT.GroupIsAlive(rangeGroupName) then
         -- Group has been killed. Clear it up and inform the owning group.
         mist.scheduleFunction(function()
-          local ownerGroup = range["owner"]
-          HOGGIT.MessageToGroup(ownerGroup:getID(), "Your range has been destroyed! Congratulations.")
-          clearRange(ownerGroup)
+          local ownerGroup = Group.getByName(owner)
+          clearRange(owner)
+          if ownerGroup and ownerGroup:isExist() then
```
Comment (Lugghawk) [inline ranges.lua:394]: I know it's not your code, but given the confusion we had tonight debugging this, can we rename `owner` to be something like `ownerGroupName` (or even something better if you think of it) which tells us what it is?

Comment (SJShaw) [reply ranges.lua:394]: Done.

File: ranges.lua (hunk 1 above)
Comment (SJShaw) [reply ranges.lua]: Fixed.

**Review by Lugghawk — APPROVED** [review]

**Pull request merged**